HLTrigger/Configuration/python/HLT_75e33/modules/siPhase2Clusters_cfi.py

```diff
@@ -5,3 +5,6 @@
     maxNumberClusters = cms.uint32(0),
     src = cms.InputTag("mix","Tracker")
 )
+
+from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
+premix_stage2.toModify(siPhase2Clusters, src = "mixData:Tracker")
```

Comment:
The following
```py
import FWCore.ParameterSet.Config as cms

from RecoLocalTracker.SiPhase2Clusterizer.phase2TrackerClusterizer_cfi import siPhase2Clusters as _siPhase2Clusters
siPhase2Clusters = _siPhase2Clusters.clone()
```
should be equivalent, and would be easier to maintain, and less error-prone.

Reply:
Sure, I can do that. Thanks for suggestion.

I tried to make very minimal change as much as possible without touching original code. I assume we have hard-copy coding around for 75e33 menu. I will fix it together with configuration when this comment, https://github.com/cms-sw/cmssw/pull/39450#issuecomment-1252829025, is confirmed.